### DISCUSSION TITLE
 ERROR  EACCES: permission denied, unlink

### DISCUSSION THREAD

**Discussion by MuttakinHasib** [pnpm/pnpm#3661] [Q&A]
@zkochan  how can I fix this issue... when I use (sudo) this `sudo pnpm add something` then it works
OS: macOS (11.4)

![Screen Shot 2021-08-11 at 11 59 47 AM](https://user-images.githubusercontent.com/44552983/128977379-4b4d0b5d-343f-4f81-be38-f3d1afde89ca.png)

**Comment by tongchong** [comment]
I have the same problem

**Reply by prodkt** [reply]
Same problem
